src/api/getBaladyiatsForDaira.js

```diff
@@ -0,0 +1,40 @@
+const MIN_DAIRA_LENGTH = 3;
+
+const getBaladyiatsForDaira = data =>
+/**
+ * Takes a wilaya daira name and returns its baladyiats
+ *
+ * @example get baladyiats for daira
+ * getBaladyiatsForDaira("ORAN")
+ *
+ * @param { String } daira The Wilaya's "daira"
+ * @returns { object[] | null } Returns All baladyiats for daira, or null
+ */
+
+  (daira) => {
+    if (!daira || daira.trim().length < MIN_DAIRA_LENGTH) {
```

Comment:
Please, last change 🤞 

```suggestion
const MIN_DAIRA_NAME_LENGTH = 3;

const getBaladyiatsForDaira = data =>
/**
 * Takes a wilaya daira name and returns its baladyiats
 *
 * @example get baladyiats for daira
 * getBaladyiatsForDaira("ORAN")
 *
 * @param { String } daira The Wilaya's "daira"
 * @returns { object[] | null } Returns All baladyiats for daira, or null
 */

  (daira) => {
    if (!daira || daira.trim().length < MIN_DAIRA_NAME_LENGTH) {
```